pkg/authorization/cache/policy.go

```diff
@@ -46,7 +46,7 @@ func NewReadOnlyPolicyCache(registry policyregistry.WatchingRegistry) readOnlyPo
 	return readOnlyPolicyCache{
 		registry:  registry,
 		indexer:   indexer,
-		reflector: *reflector,
+		reflector: reflector,
```

Comment:
I still want the `New` functions to return pointers.  That matches the other method receivers.